src/index.html

```diff
@@ -7,7 +7,52 @@
   <title>Leaderbord</title>
 </head>
 <body>
-  <p class="para">hello world</p>
-  <h1>You are welcome</h1>
+  <div class="container">
+    <h1 class="my-5">Leaderbord</h1>
+    <main class="row">
+    <section id="recent-scores" class="col-md-6 px-3">
+      <div id="recent-scores-head" class="d-flex justify-content-around">
+        <h2>Recent Scores</h2>
+        <button class="my-auto btn border border-dark">Refresh</button>
+      </div>
+
+      <div id="recent-score-body" class="px-md-3 mt-3 ">
+        <table class="table table-striped table-sm table-hover border border-dark">
+          <tbody>
+            <tr>
+              <td>Name</td>
+              <td>50</td>
+            </tr>
+            <tr>
+              <td>Name</td>
+              <td>70</td>
+            </tr>
+            <tr>
+              <td>Name</td>
+              <td>90</td>
+            </tr>
+            <tr>
+              <td>Name</td>
+              <td>80</td>
+            </tr>
+            <tr class="row-3">
+              <td>Name</td>
+              <td>100</td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
```

Comment:
- [ ] Good job making the project look similar to the wireframe provided 👍 the goal of the assignment however is to learn how to start implementing JS/ES6, right now everything visible in the browser is written in HTML. please create an arrow function with JS which will insert the information into your HTML.

Reply:
Thank you very much for your review, however the project requirements says otherwise .
below is the link to the project requirement which say that I should use plain HTML for the markup
[Project requirements ](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m1_basic_structure.md)

Reply:
This issue has been addressed well. Great job! :clap: :clap: :clap: